framework/src/media/utils/MediaUtils.cpp

```diff
@@ -68,10 +72,257 @@ audio_type_t getAudioTypeFromPath(std::string datapath)
 	} else if ((extension.compare("") == 0) || (extension.compare("pcm") == 0) || (extension.compare("raw") == 0)) {
 		medvdbg("audio type : pcm\n");
 		return AUDIO_TYPE_PCM;
+	} else if (extension.compare("wav") == 0) {
+		medvdbg("audio type : wav\n");
+ 		return AUDIO_TYPE_WAVE;
 	} else {
 		medvdbg("audio type : unknown\n");
 		return AUDIO_TYPE_INVALID;
 	}
 }
+
+bool mp3_header_parsing(unsigned char *header, unsigned int *channel, unsigned int *sample_rate, audio_format_type_t *pcmFormat)
+{
+	*pcmFormat = AUDIO_FORMAT_TYPE_S16_LE;
+	
+	unsigned char bit;
+	unsigned int mpeg_version;	
```

Comment:
Please use mp3 style.
mpegVersion, sampleRate, and so on. please check whole of yours.